build/MUX-PGOInstrument.yml

```diff
@@ -0,0 +1,64 @@
+name: $(BuildDefinitionName)_$(date:yyMM).$(date:dd)$(rev:rrr)
+jobs:
+- job: Build
+  pool:
+    vmImage: 'windows-2019'
+  timeoutInMinutes: 120
+  strategy:
+    maxParallel: 10
+    matrix:
+      Release_x86:
+        buildPlatform: 'x86'
+        buildConfiguration: 'Release'
+      Release_x64:
+        buildPlatform: 'x64'
+        buildConfiguration: 'Release'
+      Release_Arm:
```

Comment:
Are we doing PGO for arm/arm64? If not, these are unnecessary.

Reply:
The nuget package building step seems to require this. We need to add tests for arm as well shortly, so leaving this for now.

---
In reply to: [301732414](https://github.com/microsoft/microsoft-ui-xaml/pull/988#discussion_r301732414) [](ancestors = 301732414)